augerctl/command/get_command.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/etcd-io/auger/pkg/client"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type getFlagpole struct {
```

Comment:
should there be an option to limit the number of results?

Reply:
Good point, but `kubectl` doesn't have this yet, maybe we should discuss this after the PR merge.